tidb-cloud/v8.5-performance-benchmarking-with-tpcc.md

```diff
@@ -0,0 +1,208 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report for TiDB v8.5.0
+summary: Introduce the TPC-C performance test results for a TiDB Cloud Dedicated cluster with the TiDB version of v8.5.0.
+---
+
+# TiDB Cloud TPC-C Performance Test Report for TiDB v8.5.0
+
+This document provides the TPC-C performance test steps and results for a TiDB Cloud Dedicated cluster with the TiDB version of v8.5.0. This report can also be used as a reference for the performance of TiDB Self-Managed v8.5.0 clusters.
+
+## Test overview
+
+This test aims at showing the TPC-C performance of TiDB v8.5.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+### TiDB cluster
+
+The test is conducted on a TiDB cluster with the following settings:
+
+- Cluster type: [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated)
+- Cluster version: v8.5.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+    | Node type | Node size | Node quantity | Node storage |
+    |:----------|:----------|:----------|:----------|
+    | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+    | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+### Parameter configuration
+
+> **Note:**
+>
+> For TiDB Cloud, to modify the TiDB or TiKV parameters of your cluster, you can contact [PingCAP Support](/tidb-cloud/tidb-cloud-support.md) for help.
+
+The test is conducted in the following two modes:
+
+- General mode: uses default configurations
+- Tune mode: uses optimized configurations
```

Comment:
```suggestion
- Tuning mode: uses optimized configurations
```
Keep it consistent with:
https://github.com/pingcap/docs/blob/f636063ebb53d2aa9d7efb9810528b0f2e9556ac/tidb-cloud/v8.5-performance-benchmarking-with-tpcc.md?plain=1#L41